src/tests/Interop/COM/ComWrappers/Common.cs

```diff
@@ -149,6 +149,9 @@ public static AllocationCountResult CountTrackerObjectAllocations()
 
         [DllImport(nameof(MockReferenceTrackerRuntime))]
         extern public static int Trigger_NotifyEndOfReferenceTrackingOnThread();
+
+        [DllImport(nameof(MockReferenceTrackerRuntime))]
+        extern public static int IsWrapperConnected(IntPtr instance);
```

Comment:
> int

Are you just wanting to avoid the marshalling, but want `BOOL` on the native export? (Just stood out to me, since the value goes `bool` -> `BOOL` -> `int`).

Reply:
```suggestion
        extern public static int IsTrackerObjectConnected(IntPtr instance);
```